web/src/components/storage/UseDeviceMenu.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) [2025] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the Free
+ * Software Foundation; either version 2 of the License, or (at your option)
+ * any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useState } from "react";
+import { _, n_ } from "~/i18n";
+import { sprintf } from "sprintf-js";
+import {
+  Dropdown,
+  DropdownList,
+  DropdownItem,
+  DropdownGroup,
+  MenuToggleElement,
+  MenuToggle,
+  Divider,
+} from "@patternfly/react-core";
+import { MenuHeader } from "~/components/core";
+import MenuDeviceDescription from "./MenuDeviceDescription";
+import { useAvailableDevices, useConfigModel, useDrives } from "~/queries/storage";
+import { deviceLabel } from "~/components/storage/utils";
+
+export default function UseDeviceMenu() {
```

Comment:
I know naming is hard as hell, but I'd avoid `UseDeviceMenu`. In general, I'd avoid naming components with verbs as prefix, but specially `Use` in the React world is too couple to hooks IMHO.

Reply:
What about `DevicesMenu`? More consistent whith the rest of name until now which looks like Thing plus Group when needed, as `ProposalPage`, `ConfigEditorMenu`, etc.